Blog/templates/base_generic.html

```diff
@@ -20,16 +20,29 @@
 <div class="nav-bar">
 
     {% block sidebar %}
-        <ul class="h-menu bg-blue fg-white fixed-top">
-            <li><a href="{% url 'index' %}"><span class="mif-home icon"></span> Home</a></li>
-            <li><a href="{% url 'posts' %}"><span class="mif-file-text icon"></span> All Posts</a></li>
-            <li><a href="{% url 'users' %}"><span class="mif-users icon"></span> All Bloggers</a></li>
-            {% if user.is_authenticated %}
-                <li><a href="{% url 'logout' %}"><span class="mif-lock icon"></span> Logout</a></li>
-            {% else %}
-                <li><a href="{% url 'login' %}"><span class="mif-key icon"></span> Login</a></li>
-            {% endif %}
-        </ul>
+        {#        <ul class="h-menu bg-blue fg-white fixed-top">#}
+        {#            <li><a href="{% url 'index' %}"><span class="mif-home icon"></span> Home</a></li>#}
+        {#            <li><a href="{% url 'posts' %}"><span class="mif-file-text icon"></span> All Posts</a></li>#}
+        {#            <li><a href="{% url 'users' %}"><span class="mif-users icon"></span> All Bloggers</a></li>#}
+        {#            {% if user.is_authenticated %}#}
+        {#                <li><a href="{% url 'logout' %}"><span class="mif-lock icon"></span> Logout</a></li>#}
+        {#            {% else %}#}
+        {#                <li><a href="{% url 'login' %}"><span class="mif-key icon"></span> Login</a></li>#}
+        {#            {% endif %}#}
+        {#        </ul>#}
+        <div class="app-bar-expand-md bg-blue fg-white" data-role="appbar">
```

Comment:
How is this change related to the commenting feature? If it is not related, it should be put into a separate comment where the commit message must contain the reason for this change.